netcdftime/_netcdftime.pyx

```diff
@@ -669,15 +678,21 @@ def DateFromJulianDay(JD, calendar='standard'):
     if calendar in 'proleptic_gregorian':
```

Comment:
In looking at this logic, I noticed that as written this branch of the if-statement is used for both `calendar='proleptic_gregorian'` and `calendar='gregorian'`.  Therefore, while the next branch is `elif calendar in ('standard', 'gregorian')` it will never be used in the case of `calendar='gregorian'`.  Was this intended?

In other words, should this line be `if calendar == 'proleptic_gregorian'`?